graylog2-server/src/test/resources/org/graylog2/plugin/inputs/transports/util/server.key.pem.pkcs8.protected

```diff
@@ -0,0 +1,30 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
For completeness you could also add a PBE1 encrypted key, like here:

https://github.com/Graylog2/graylog2-server/blob/master/graylog2-server/src/test/resources/org/graylog2/shared/security/tls/key-enc-pbe1.p8